Packs/Rapid7InsightVMCloud/Integrations/Rapid7InsightVMCloud/Rapid7InsightVMCloud.yml

```diff
@@ -0,0 +1,107 @@
+  display: Server URL (e.g., https://example.net)
+  name: base_url
+  required: true
+  type: 0
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: InsightVM is a Vulnerability Management Tool which Scan your Network, Eliminate Vulnerabilities, Track and Communicate progress.
+display: Rapid7 InsightVM Cloud
+name: Rapid7 InsightVM Cloud
+script:
+  commands:
+  - arguments:
+    - description: The identifier of the asset to retrieve the details for.
+      name: asset_id
+      required: true
+    description: Returns the assessment and details of an asset (specified by id). Only assets which the caller has access to can be returned.
+    name: insightvm-cloud-get-asset
+  - arguments:
+    - description: The identifier of the scan.
+      name: scan_id
+      required: true
+    description: Retrieves the scan with the specified identifier.
+    name: insightvm-cloud-get-scan
+  - arguments:
+    - defaultValue: "0"
+      description: The index of the page to retrieve.
+      name: page
+    - defaultValue: "10"
+      description: The number of records per page to retrieve.
+      name: size
+    description: Retrieves a page of scan engines.
+    name: insightvm-cloud-get-scan-engines
+  - arguments: []
+    description: Returns an indicator of the health of the API.
+    name: insightvm-cloud-get-health-check
+  - arguments:
+    - description: Search criteria for filtering assets returned.
+      name: hostname
+      required: true
+    - defaultValue: "0"
+      description: The index of the page (zero-based) to retrieve.
+      name: page
+    - defaultValue: "10"
+      description: The number of records per page to retrieve.
+      name: size
+    description: Returns the inventory, assessment, and summary details for a page of assets.
+    name: insightvm-cloud-search-assets
+  - arguments:
+    - defaultValue: "0"
+      description: The index of the page (zero-based) to retrieve.
+      name: page
+    - defaultValue: "10"
+      description: ' The number of records per page to retrieve.'
+      name: size
+    description: Returns the details for sites
+    name: insightvm-cloud-last-sites
+  - arguments:
+    - defaultValue: "0"
+      description: The index of the page to retrieve
+      name: page
+    - defaultValue: "10"
+      description: The number of records per page to retrieve
+      name: size
+    - description: ' query to search vulnerabilities.'
+      name: query
+      required: true
+    description: Returns all vulnerabilities that can be assessed.
+    name: insightvm-cloud-search-vulnerabilities
+  - arguments:
+    - description: The identifiers of the assets to scan
+      name: asset_id
+      required: true
+    - description: The name of the scan
+      name: name
+    description: Starts a scan.
+    name: insightvm-cloud-start-scan
+  - arguments:
+    - description: The identifiers of the assets to stop scan
```

Comment:
Is this input for multiple inputs? If so, it should be:
A comma-separated list of asset IDs used to stop the scan.
Otherwise, if this is a single input, then it should be:
The identifier of the asset used to stop the scan.

Reply:
I have edited the suggested change.